dataline-workers/src/main/java/io/dataline/workers/DefaultSyncWorker.java

```diff
@@ -62,14 +61,8 @@ public DefaultSyncWorker(
   public OutputAndStatus<StandardSyncOutput> run(StandardSyncInput syncInput, Path workspacePath) {
     long startTime = System.currentTimeMillis();
 
-    final StandardTapConfig tapConfig = new StandardTapConfig();
-    tapConfig.setSourceConnectionImplementation(syncInput.getSourceConnectionImplementation());
-    tapConfig.setStandardSync(syncInput.getStandardSync());
-
-    final StandardTargetConfig targetConfig = new StandardTargetConfig();
-    targetConfig.setDestinationConnectionImplementation(
-        syncInput.getDestinationConnectionImplementation());
-    targetConfig.setStandardSync(syncInput.getStandardSync());
+    final StandardTapConfig tapConfig = WorkerUtils.syncToTapConfig(syncInput);
```

Comment:
just a thought. I wonder if we can remove the `Standard` prefix. and for non-standard, then we add the prefix

Reply:
yeah. i think think the standard concept is getting pretty unwieldy.

Reply:
made an issue for this. not going to tackle it here. https://github.com/datalineio/dataline/issues/132